pages/content/amp-dev/documentation/guides-and-tutorials/integrate/story-interactive.md

```diff
@@ -0,0 +1,71 @@
+
+Elements that require image URLs such as amp-story-interactive-results can use already existing workflows to upload and show images.
+
+Multi-state elements such as amp-story-interactive-results should allow users to change the values on each state independently either inline or on a side panel, but it's a good idea for users to preview each state as they are working on them, so having a preview selector could be a good idea.
+
+Previews cannot always visualize all the fields (such as confetti effects), but tools can show a preview of the animation only played when users change the values associated with it. Remember that some elements allow all options to have a confetti emoji, while others limit the confetti to certain options. 
+
+[tip type="note"]
+Adding animations to the previews can also allow `animate-in` effects in the interactive elements, which are a great way to customize the experience further.
+[/tip]
+
+Some elements have additional configuration for their options, which will require custom code in the UI to input (eg: what option is correct in a quiz), but most elements share to some extent all the content attributes, so reusing the code and UI is a good idea.
+
+### Style management
+
+Most elements support the same set of styling options, either through CSS variables or element attributes. All tools have different ways of styling elements, so they will provide a unique approach to customizing them. Tools that focus on consistent styling could incorporate an existing color palette for a story into all components at the same time, since applying the same set of CSS values to all the elements yield consistent results. Tools that focus on individual customization of elements may mave more refined controls over all CSS variables that are exposed for each element.
+
+In terms of actual implementations, it's a good idea to give users control over the theme and chip-style of the elements, and ideally also allow them to control the accent color and prompt alignment. Variables such as prompt text and prompt background are interesting for users since they allow prompt backgrounds to be gradients, and they allow prompt text to be colored to match better the story design cues and improve contrast.
+
+Tools can go above and beyond to give extra customization options for creators. Many interesting designs can be implemented with the `chip-style="transparent"` attribute. For inspiration check out the [amp-story-interactive-results Codepen](https://codepen.io/mszylkowski/pen/abNWzdb) which contains creative designs for transparent styles. Those require extra CSS rules to be modified for the elements, such as the actual element's background, border (and border-radius), box-shadow, backdrop-filter, etc. When enabling extra options, keep in mind use-cases for them, as many CSS rules will not work on the elements (eg: font-family).
```

Comment:
What's interesting in this part is the design options and guidance you're giving. How much control a tool gives to its users depends on so much (how savvy are they, does the tool make money from selling templates / customization options, does the tool want to create a design brand/identity, etc), I don't think it's our business to tell them what they should do.